src/styles/stars.scss

```
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+
+  &__star {
+    height: $star-size;
+    width: $star-size;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--1 :first-child {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--2 :nth-child(-n + 2) {
+    background-image: url(../images/star-active.svg);
+  }
```

```suggestion
  &--1 :first-child,
  &--2 :nth-child(-n + 2) {
    background-image: url(../images/star-active.svg);
  }
```
combine it with a comma